integration_test/images_test.go

```diff
@@ -137,32 +142,36 @@ var _ = Describe("Image integration tests", func() {
 		AfterEach(func() {
 			assistedServer.Close()
 			imageServer.Close()
+			Expect(os.Remove(isoFile)).To(Succeed())
```

Comment:
I think `RemoveAll` here so it doesn't fail if the image didn't get written for some reason.